recipes/metview-batch/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "Metview" %}
+{% set version = "5.7.5" %}
+
+package:
+  name: metview-batch
+  version: {{ version }}
+
+source:
+  url: https://confluence.ecmwf.int/download/attachments/3964985/{{ name }}-{{ version }}-Source.tar.gz
+  sha256: 6362a08a071b5b3b5e0c6f23bdfeebe81341fe30a7a687d983ae2acf1afac57a
+  patches:
+    - 0001-Use-find_package-instead-of-find_library-for-threads.patch
+    - 0006-Pass-CPP-to-rpcgen.patch
+    - 0007-Allow-to-build-without-Fortran-compiler.patch
+# patch 0007 will be redundant with Metview 5.8.0
+
+build:
+  number: 0
+  skip: True  # [win]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - libcurl
+    - eccodes
+    - magics-metview-batch
+    - boost
+    - flex
+    - bison
+    - libnetcdf
+    - proj
+    - gdbm
+    - fftw
+    - _openmp_mutex  # [linux]
```

Comment:
See https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp on how to properly indicate `openmp`.